src/app/validators/custom-validators.ts

```diff
@@ -13,7 +13,7 @@ export class CustomValidators {
     if (!ac.value) {
       return null;
     }
-    return (ac.value > 0) ? null : { invalidPositive : true };
```

Comment:
This changes the definition of this function in an unexpected way.  Let's try using the `integerValidator` instead.

Reply:
@paulbert  **integerValidator** is not working as we have thought in this issue. So can you suggest me another validator which works in this condition.